src/main/java/com/ibm/watson/modelmesh/ModelMeshApi.java

```diff
@@ -783,8 +784,13 @@ public void onHalfClose() {
                     call.close(status, emptyMeta());
                     Metrics metrics = delegate.metrics;
                     if (metrics.isEnabled()) {
-                        metrics.logRequestMetrics(true, methodName, nanoTime() - startNanos,
-                                status.getCode(), reqSize, respSize);
+                        if (isVModel) {
+                            metrics.logRequestMetrics(true, methodName, nanoTime() - startNanos,
+                                    status.getCode(), reqSize, respSize, "", Iterables.toString(modelIds));
```

Comment:
`Iterables.toString()` I think will produce a comma-separated list of the contained strings. That's not what we want for the label. Passing mutliple model or vmodel ids is kind of a niche case but really we should iterate over them and log separate metrics for each.

Reply:
resolved, looping over the list and logging metrics separately